exchanges/order/limits.go

```diff
@@ -0,0 +1,364 @@
+package order
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"sync"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+)
+
+var (
+	// ErrExchangeLimitNotLoaded defines if an exchange does not have minmax
+	// values
+	ErrExchangeLimitNotLoaded = errors.New("exchange Limits not loaded")
+	// ErrPriceExceedsMin is when the price is lower than the minimum price
+	// Limit accepted by the exchange
+	ErrPriceExceedsMin = errors.New("price exceeds minimum Limit")
+	// ErrPriceExceedsMax is when the price is higher than the maximum price
+	// Limit accepted by the exchange
+	ErrPriceExceedsMax = errors.New("price exceeds maximum Limit")
+	// ErrPriceExceedsStep is when the price is not divisible by its step
+	ErrPriceExceedsStep = errors.New("price exceeds step Limit")
+	// ErrAmountExceedsMin is when the amount is lower than the minimum amount
+	// Limit accepted by the exchange
+	ErrAmountExceedsMin = errors.New("amount exceeds minimum Limit")
+	// ErrAmountExceedsMax is when the amount is higher than the maximum amount
+	// Limit accepted by the exchange
+	ErrAmountExceedsMax = errors.New("amount exceeds maximum Limit")
+	// ErrAmountExceedsStep is when the amount is not divisible by its step
+	ErrAmountExceedsStep = errors.New("amount exceeds step Limit")
+	// ErrNotionalValue is when the notional value does not exceed currency pair
+	// requirements
+	ErrNotionalValue = errors.New("total notional value is under minimum Limit")
+	// ErrMarketAmountExceedsMin is when the amount is lower than the minimum
+	// amount Limit accepted by the exchange for a market order
+	ErrMarketAmountExceedsMin = errors.New("market order amount exceeds minimum Limit")
+	// ErrMarketAmountExceedsMax is when the amount is higher than the maximum
+	// amount Limit accepted by the exchange for a market order
+	ErrMarketAmountExceedsMax = errors.New("market order amount exceeds maximum Limit")
+	// ErrMarketAmountExceedsStep is when the amount is not divisible by its
+	// step for a market order
+	ErrMarketAmountExceedsStep = errors.New("market order amount exceeds step Limit")
+
+	errCannotValidateAsset         = errors.New("cannot check Limit asset not loaded")
```

Comment:
Grammatical issue for this one and a few others below

```suggestion
	errCannotValidateAsset         = errors.New("cannot check limit, asset not loaded")
```